pyproject.toml

```diff
@@ -19,6 +19,7 @@ classifiers = [
 dependencies = [
     "PyCryptodome",
     "html2text",
+    "litellm",
     "numpy",
     "openai>=1",
```

Comment:
I think we want to drop this as a primary dependency now too

Reply:
Yeah I like the idea. Note that LiteLLM depends on `openai`, so it's not gonna lighten the load overall, but I agree it postures _our_ code to use LiteLLM for everything